src/khoj/database/models/__init__.py

```diff
@@ -79,7 +79,7 @@ class Type(models.TextChoices):
     enabled_trial_at = models.DateTimeField(null=True, default=None, blank=True)
 
 
-class OpenAIProcessorConversationConfig(BaseModel):
+class AiModelApi(BaseModel):
```

Comment:
It seems to me that the `ModelType` field in `ChatModelOptions` is better served here than in the other. The AI model API will only fit one of those types, and it's better defined as part of the provider, rather than the chat model. Might be a helpful bit of refactor to include with this change, but may also be out of scope.